### DISCUSSION TITLE
存储驱动OneDrive APP，授权啥的步骤全部按要求操作了，也都成功了，最后不能显示文件夹

### DISCUSSION THREAD

**Discussion by FoundZiGu** [AlistGo/alist#4048] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [x] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

V3.15.0

### Driver used / 使用的存储驱动

OneDrive APP

### Describe the bug / 问题描述

存储驱动OneDrive APP，授权啥的步骤全部按要求操作了，也都成功了，最后不能显示文件夹
报错：failed get objs: failed to list objs: User not found

### Reproduction / 复现链接

https://cloud.zigu.me/

### Logs / 日志

[![pp5T3tO.png](https://s1.ax1x.com/2023/04/05/pp5T3tO.png)](https://imgse.com/i/pp5T3tO)

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by youtehub** [comment]
直接用webdav挂载呗，进入onedrive页面可以直接找到地址

**Comment by chengnan049** [comment]
我这边可以复现，环境如下：
账号：个人号拼车 365 家庭版
Alist Version: 3.30.0
我自己改过很多邮箱前缀不知道是不是这个原因，我的主名是C开头的，不过 `Primary domain` 中显示的不是这个别名，是老的别名，而且后缀是 `onmicrosoft.com` ，且字段中没有 `_` 及 `.`

**Comment by cmubioinformatics** [comment]
同问题 个人账号 家庭版拼车的主账号
failed get objs: failed to list objs: User not found

**Comment by Arturia169** [comment]
解决了吗 我也是这个问题

**Reply by chengnan049** [reply]
换隔壁OpenList吧

**Reply by Arturia169** [reply]
> 换隔壁OpenList吧

行我去瞅瞅
